examples/examples-by-storage/demos.py

```diff
@@ -10,58 +10,57 @@ def list_domains(modelstore: ModelStore):
         print(f"\t  Domain: {domain}")
 
 
-def list_models_in_domain(modelstore: ModelStore, model_domain: str):
+def list_models_in_domain(modelstore: ModelStore, model_domain: str) -> list:
     print(f"✅  Listing models for {model_domain}:")
     model_ids = modelstore.list_models(domain=model_domain)
     for model_id in model_ids:
         print(f"\t  Domain: {model_domain} has model with id={model_id}")
+    return model_ids
```

Comment:
These are small demo-like functions to show how to use different parts of `modelstore`. I've tweaked them so that I can also run some assertions on their outputs